domains/sajidportfolio.json

```
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "Sajid-tech",
+        "email": "s.khan9430319425@gmail.com"
+    },
+    "record": {
+        "CNAME": "https://sajid-tech.github.io"
```

`CNAME` should not contain protocol
```suggestion
        "CNAME": "sajid-tech.github.io"
```

---

updated